files/common/etc/systemd/journald.conf.d/override.conf

```diff
@@ -1,8 +1,15 @@
+[Journal]
 #
 # Configure journald to persistently keep logs, so that we can inspect
 # them after a reboot has already occurred. This helps debugging, and
 # also can make the collection of support bundles more useful.
 #
-[Journal]
 Storage=persistent
 SystemMaxUse=2.5G
+#
+# Disable the rate limiting feature, so that we don't end up truncating
+# the logs of noisy services. The truncatation may otherwise inhibit
+# root cause analysis of customer problems.
+#
+RateLimitIntervalSec=0
+RateLimitBurst=0
```

Comment:
Reading the comments for DLPX-61397, it seems to me like we were more inclined to increase the burst limit rather than eliminating it all-together. What made you decide to go for the latter?

Reply:
This PR was opened prior to that discussion. I haven't updated this to reflect those comments yet.